index.html

```diff
@@ -2122,41 +2113,39 @@
 destruction.
```

Comment:
Yikes! What parts of this do we actually need to list, and what parts could we simplify?

Reply:
Left for discussion in #375.